Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -82,11 +85,15 @@ private void setUpMotorTeam(XCANTalon master) {
         master.configPeakOutputVoltage(12, -6);
 
         master.setProfile(0);
-        master.setF(.099);
         updateMotorPidValues(master);
         master.setControlMode(TalonControlMode.Speed);
-
         master.set(0);
+        
+        follower.setProfile(0);
+        follower.setControlMode(TalonControlMode.Follower);
+        follower.configNominalOutputVoltage(0,  -0);
+        follower.configPeakOutputVoltage(12, -6);
```

Comment:
I believe the follower needs to spin backwards relative to the master. That should be configured, and additionally these values may need to be swapped. Or they could probably just be removed, given that the master constraints should apply here too.